mono/mini/mini-runtime.h

```diff
@@ -174,6 +174,18 @@ typedef struct MonoDebugOptions {
 	gboolean dyn_runtime_invoke;
 	gboolean gdb;
 	gboolean lldb;
+
+	/*
+	 * With LLVM codegen, this option will cause methods to be called indirectly through the
+	 * PLT (As they are in other FullAOT modes, without LLVM). The indirection allows 
```

Comment:
I think this should be 'enable this to debug problems with direct calls in llvm'.